test/integration_tests.jl

```diff
@@ -0,0 +1,150 @@
+# You should set up a Git repo with an example Julia package for the integration tests.
+# You should create a bot user GitHub account, and give the bot user GitHub account push access to the testing repo.
+# You should generate a personal access token (PAT) for the bot user GitHub account.
+# You should add the PAT as an encrypted Travis CI environment variable named `BCBI_TEST_USER_GITHUB_TOKEN`.
+# You should generate an SSH deploy key. Upload the public key as a deploy key for the testing repo.
+# You should make the private key an encrypted Travis CI environment variable named `COMPATHELPER_PRIV`. Probably you will want to Base64-encode it.
+
+@testset "GitHub Integration Test" begin
+    GLOBAL_PR_TITLE_PREFIX = Random.randstring(8)
+    COMPATHELPER_INTEGRATION_TEST_REPO = ENV["COMPATHELPER_INTEGRATION_TEST_REPO"]
+    TEST_USER_GITHUB_TOKEN = ENV["BCBI_TEST_USER_GITHUB_TOKEN"]
+
+    url = "https://github.com/$(COMPATHELPER_INTEGRATION_TEST_REPO)"
+    api = GitHub.GitHubAPI(; token=token, url=url)
```

Comment:
the url argument here is to point to a different api url, I don't think you put the repo url in here, also you wanna plug in the token
```suggestion
    api = GitHub.GitHubAPI(; token=TEST_USER_GITHUB_TOKEN)
```